### PR TITLE
feat: introducing whoami dedicated route

### PR DESCRIPTION
**Pull request by rebelchris**
Introducing a whoami call that the gateway can use to retrieve user profile from the API.

WT-296

### CODE REVIEW

File: src/common/users.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export interface User {
   email: string;
   name: string;
   image: string;
+  infoConfirmed: boolean;
```
Comment (kirillkurko) [inline src/common/users.ts:21]: Maybe it can be just `confirmed` or `verified`, what do you think?

**Review by kirillkurko — APPROVED** [review]

File: src/common/users.ts (hunk 1 above)
Comment (idoshamun) [reply src/common/users.ts:21]: We must keep it `infoConfirmed` for backwards compatibility

**Review by idoshamun — APPROVED** [review]

**Comment by rebelchris** [comment]
@idoshamun I just went over all gatewayUser fixtures and changed them to be loaded from 1 origin.
This is a bit more maintable for when we add/remove fields later on.

Also this one seems like we can already merge, doesn't introduce anything breaking for the current version.

**Review by idoshamun — APPROVED** [review]
I agree, this shouldn't cause any breaking changes. We can safely merge

**Pull request merged**